File: solution/ServerExam/Entity/ManagerEntity.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace ServerExam.Entity
{
    public class ManagerEntity
    {
        [Key]
        [JsonPropertyName("ID")]
        public int ID { get; set; }

        /// <summary>
        /// 账号
        /// </summary>
        [JsonPropertyName("Account")]
        public string Account { get; set; }

        /// <summary>
        /// 密码
        /// </summary>
        [JsonPropertyName("PWD")]
        public string PWD { get; set; }

        /// <summary>
        /// 名字
        /// </summary>
        [JsonPropertyName("Name")]
        public string Name { get; set; }

        /// <summary>
        /// 状态 1正常 2禁用
        /// </summary>
        [JsonPropertyName("State")]
        public int State { get; set; }

        /// <summary>
        /// 权限
        /// </summary>
        [JsonPropertyName("Permission")]
        public int Permission { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        [JsonPropertyName("CreateTime")]
        public int CreateTime { get; set; }

        /// <summary>
        /// 更新时间
        /// </summary>
        [JsonPropertyName("UpdateTime")]
        public int UpdateTime { get; set; }

        /// <summary>
        /// Token
        /// </summary>
        [JsonPropertyName("Token")]
        public string Token { get; set; }

        public ManagerEntity()
        {
            this.ID = 0;
            this.Account = "";
            this.PWD = "";
            this.Name = "";
            this.State = 0;
            this.Permission = 0;
            this.CreateTime = 0;
            this.UpdateTime = 0;
            this.Token = "";
        }
    }
}

File: solution/Entity/ManagerEntity.py
from Entity.BaseEntity import *


# 管理员
class ManagerEntity(BaseEntity, BaseORM):
    __tablename__ = 'Manager'

    Account: str = Column(String(128), comment='账号', default='none')
    PWD: str = Column(String(128), comment='密码', default='none')
    Name: str = Column(String(128), comment='名称', default='none')
    State: int = Column(INTEGER, comment='状态 1正常 2禁用', default=0)
    Permission: int = Column(INTEGER, comment='权限 9 ~ 1 从高到低', default=0)
    UpdateTime: int = Column(INTEGER, comment='更新时间', default=int(time()))
    Token: str = Column(String(128), comment='Token', default='none')

    def __init__(self):
        super().__init__()